handlers/association/association_handler.go

```
@@ -0,0 +1,973 @@
+	err = h.db.QueryRow("SELECT name FROM associations WHERE id_associations = $1", associationID).Scan(&associationName)
+	if err != nil {
+		utils.LogMessage(utils.LevelError, "Failed to get association name")
+		utils.LogFooter()
+		return c.Status(fiber.StatusInternalServerError).JSON(fiber.Map{
+			"error": "Failed to get association information",
+		})
+	}
+
+	roleName := fmt.Sprintf("%s_respo", strings.ToLower(strings.ReplaceAll(associationName, " ", "")))
+	var isRespo bool
+	err = h.db.QueryRow(`
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)